### ISSUE TITLE
Systemd service

### ISSUE THREAD

**Issue by ghost** [Cygn/pulseaudio-dlna#20]
Hello ! 

Is it possible to make it a systemd service so that your program can be started on start-up ? 

Thanks

**Comment by Cygn** [comment]
Hello,
It's not my program per-se, I'm just packaging it for Fedora & co.  I won't do any dev, but I'm open to diverge from upstream (which is  stale) if someone pushes new functionality as a patch. 
Anyway, there was some discussion in the past around autostart, you can browse [here](https://github.com/masmu/pulseaudio-dlna/issues/276). The problem is to make the service run only after pulseaudio and a working network connection - feel free to play around.
